io/disk/htx_block_devices.py

```diff
@@ -152,8 +152,13 @@ def install_htx_rpm(self):
             if distro_pattern in htx_rpm]
         distro_specific_htx_versions.sort(reverse=True)
         self.latest_htx_rpm = distro_specific_htx_versions[0]
-
-        if process.system(f"rpm -ivh --nodeps {self.rpm_link}{self.latest_htx_rpm} --force",
+        tmp_dir_rpm = "/tmp/" + self.latest_htx_rpm
+        cmd = "curl -k %s/%s -o %s" % (self.rpm_link,
+                                       self.latest_htx_rpm,
+                                       tmp_dir_rpm)
+        if process.system(cmd, shell=True, ignore_status=True):
+            self.cancel(f"Dowload of hrx rpm {self.latest_htx_rpm} from rpm link {self.rpm_link}")
```

Comment:
download of htx failed from...

Reply:
Yeah you are true, Corrected typos and made the message meaningful.